pkg/apis/submarinerconfig/v1alpha1/types.go

```diff
@@ -176,9 +176,9 @@ type ManagedClusterInfo struct {
 	// Region represents the cloud region of the managed cluster.
 	// +optional
 	Region string `json:"region,omitempty"`
-	// InfraId represents the infrastructure id of the managed cluster.
+	// InfraID represents the infrastructure id of the managed cluster.
 	// +optional
-	InfraId string `json:"infraId,omitempty"`
+	InfraID string `json:"infraID,omitempty"`
```

Comment:
We shouldn't change rendered representation, just the Go field.

```suggestion
	InfraID string `json:"infraId,omitempty"`
```